opennms-doc/guide-admin/src/asciidoc/text/provisioning/directed-discover.adoc

```diff
@@ -0,0 +1,80 @@
+
+// Allow GitHub image rendering
+:imagesdir: ../../../images
+
+[[directed-discovery]]
+=== Directed Discovery
+
+Directed discovery is the process of manually adding nodes to {opennms-product-name}.
+There are three ways to do this:
+
+* xref:requisition-add-nodes[directly in the requisition UI]
+* xref:quick-add-node [quick add node]
+* manually adding an interface
+
+Make sure you complete the tasks in the xref:getting-started.adoc[Getting Started] section before adding nodes. 
+
+[[requisition-add-nodes]]
+==== Add Nodes through the Requisition UI
+
+Before adding nodes to a requisition, you must xref:requisition-create[create a requisition].
+
+. In the web UI, click the gear icon in the top right. 
+. In the *Provisioning* area, choose *Manage Provisioning Requisition*.
+. Click the edit icon beside the requisition you want to add nodes to. 
+. Click *Add Node*. 
+.. {opennms-product-name} auto-generates the foreign ID used to identify this node. 
+. Fill out information in each of the tabs and click *Save*. 
++
+* basic information (node label, auto-generated foreign ID, location)
+* path outage (configure network path to limit notifications from nodes behind other nodes)
+* interfaces (add interface IP addresses and services)
+* assets (pre-defined metadata types)
+* categories (label/tag for type of node, e.g., routers, production, switches)
+* meta-data (customized asset information)
+
+. Repeat for each node you want to add. 
+. Click *Return* to view the list of nodes you have added. 
+. Click *Synchronize* to provision them to the {opennms-product-name} database. 
+
+[[quick-add-node]]
+==== Add a Single Node to a Requsition
+
+The quick-add node feature allows you to add a single node to an existing provisioning requisition, and set the community string at the same time the node is provisioned.  
+
+When you configure an individual IP address within an existing range, {opennms-product-name} will split the range into two, and add a specific entry for the IP address. 
+See xref:quick-add-xml[Quick-Add Node Configuration] for the resulting XML configuration.
+
+To use the quick-add node feature, follow these steps:
+
+. Make sure you have xref:requisition-create[created a requisition]. 
+. In the web UI, click the *+* sign beside the gear icon.
+. Specify the requisition to which you want to add the node. 
+
++
+IMPORTANT: This operation overrides any un-synchronized modifications in the selected requisition.
+
+. Add the IP address and node label. 
+. Fill in optional information in the apporpriate fields:
+
++
+* SNMP parameters (apply only to the node being requisitioned)
+* surveillance category membership (like a tag for the type of node, e.g., router, production, switch)
+* CLI authentication parameters (used only if one or more provisioning adapters are configured to use them)
```

Comment:
IMHO, the "Quick Add Node", which is where I think the above screenshot is coming from, should be removed as a whole. That existed because a customer requested that in the past (and I don't believe it is being used, being honest), so I'm not against or in favor, so I'll you decide.

Reply:
How do we track requests like this (to remove a feature in the software)? In the interim, until it is removed, we can add something to the documentation to the effect of "Quick Add Node" is being deprecated. We do not recommend using it to provision a node." Thoughts?

Reply:
+1 to getting rid of Quick Add Node for the reasons @agalue named. Not sure how to track that sort of thing.

Tech detail: since it's now possible to specify a foreign-source name as a parameter in a newSuspect event (which is what the Manually Add an Interface flow produces), the last remaining bit of value in Quick Add Node could be recovered by exposing this parameter in Manually Add an Interface.